### PR TITLE
[WIP] Add transpiling and linting

### PR DESCRIPTION
**Pull request by JerryLopez**
@NickTikhonov Finally got sometime to get started on this.

## Transpiling

For transpiling I am using Babel and Webpack, which builds a bundle file called `hyperline.js` located in `dist/`. 

I ran into an issue after transpiling and bundling, but haven't had a chance to troubleshoot. Below is a screenshot of what I'm experiencing. If you have any suggestions feel free to shoot them my way.

screen shot 2016-07-30 at 5 56 31 pm

## Linting

I haven't added any linting yet, but considered using [ESLint](http://eslint.org/).

### CODE REVIEW

**Comment by NickTikhonov** [comment]
@JerryLopez - strange! I'll take a look at it later today - looks like there's a bug in how the stats are wired in. I'm also all for ESLint - I'm using it in another project and it's great! We should add it early on so we can agree on a style standard before this plugin gets too big. Overall great work! 🎉 🎉

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
 npm-debug.log
+dist
```
Comment (NickTikhonov) [inline .gitignore]: Will HyperTerm automatically run `build

**Comment by timneutkens** [comment]
Hey @JerryLopez found why it shows these weird characters. You should add `target: 'node'` to `webpack.config.js` 👍

File: package.json [hunk 2]
```diff
@@ -2,17 +2,33 @@
   "name": "hyperline",
   "version": "0.2.0",
   "description": "Handy status line for HyperTerm",
-  "keywords": ["hyperterm"],
-  "main": "index.js",
+  "keywords": [
+    "hyperterm"
+  ],
+  "main": "dist/hyperline.js",
   "scripts": {
```
Comment (timneutkens) [inline package.json]: This should be removed since there is a new `"scripts"` node

**Comment by timneutkens** [comment]
Took the time to just pull request it myself ;) https://github.com/JerryLopez/hyperterm-hyperline/pull/1

**Comment by JerryLopez** [comment]
@timneutkens thanks a lot for fixing that 😁

**Comment by timneutkens** [comment]
Should have fixed those merge conflicts too :joy:

**Comment by timneutkens** [comment]
Not related to the pull request.. @JerryLopez @NickTikhonov are you guys on the [ZEIT slack channel](https://zeit.chat/)?

**Comment by JerryLopez** [comment]
@timneutkens nope I'm not, but I'm joining now 👍 Are you on there?

**Comment by timneutkens** [comment]
But ofcourse 😄

**Comment by JerryLopez** [comment]
@timneutkens done!

**Comment by NickTikhonov** [comment]
Great work @JerryLopez ! Definitely joining the Zeit Slack - thanks for letting me know it exists @timneutkens 👐

**Pull request merged**